src/devices/tapestry.ts

```diff
@@ -0,0 +1,21 @@
+import {Definition} from '../lib/types';
```

Comment:
Please also add this file to https://github.com/Tapestry-Home/zigbee-herdsman-converters/blob/master/src/devices/index.ts